opba-protocols/xs2a-protocol-tests/xs2a-bdd-wiremock/src/test/java/de/adorsys/opba/protocol/xs2a/tests/e2e/wiremock/WiremockE2EXs2aProtocolTest.java

```diff
@@ -529,4 +529,29 @@ void testAccountsListWithConsentUsingRedirectWithoutIpAddress() {
                 .user_anton_brueckner_provided_initial_parameters_to_list_accounts_with_all_accounts_consent_and_gets_202();
     }
 
+    @ParameterizedTest
+    @EnumSource(Approach.class)
+    void testAccountsListWithConsentUsingRedirectWithMissingPsuId(Approach expectedApproach) {
+        given()
+                .redirect_mock_of_sandbox_for_anton_brueckner_accounts_running()
```

Comment:
Additionally, suggest you modify Wiremock configuration so that it loads files from different directories - as you are going to require multiple fixtures. This can be done by creating a temporary directory and loading all fixtures into it